### DISCUSSION TITLE
Header Loading is not Showing on any action,

### DISCUSSION THREAD

**Discussion by shakeel2717** [Power-Components/livewire-powergrid#1421] [Q&A]
IN recent version, whenever i use editOnClick or action button, the header show loading icon, but in current version, the loading bar is not showing, everything working fine, but on action click, no loading showing, it's looks like the button is not clicked, and with this confusions, users clicking button too many time, 

i also read the docs
also try with this way

return [
            Exportable::make('export')
                ->striped()
                ->type(Exportable::TYPE_XLS, Exportable::TYPE_CSV),
            Header::make()->showSearchInput(),
            Footer::make()
                ->showPerPage()
                ->showRecordCount(),
        ];

and also with this way

return [
            Exportable::make('export')
                ->striped()
                ->type(Exportable::TYPE_XLS, Exportable::TYPE_CSV),
            Header::make()->showSearchInput()->withoutLoading(),
            Footer::make()
                ->showPerPage()
                ->showRecordCount(),
        ];

**Comment by luanfreitasdev** [comment]
Hello. This is a behavior caused by withoutLoading. If you take it off, does it work? Open an issue with details

**Reply by kurniasihmartha** [reply]
I have the same problem. I dont use withoutLoading(). but nothing showed up in the header.
For additional information, I use bootstrap-5 theme
